easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -16,9 +16,10 @@ sources = [SOURCE_TAR_GZ]
 
 configopts = [
     # default build
-    '--with-shared --enable-overwrite',
+    '--with-shared --enable-overwrite --with-termlib',
```

Comment:
I only had to do this to be able to build lua at the `dummy` level, it was never an issue for me, it was an issue for the tests.